tests/FertiScan/analysis_formatted.json

```diff
@@ -0,0 +1,187 @@
+{
+    "company": {
+        "id": "69c752bd-41cc-4595-88c7-b506592caf34",
+        "name": "GreenGrow Fertilizers Inc.",
+        "address": "123 Greenway Blvd, Springfield IL 62701 USA",
+        "website": "http://www.greengrowfertilizers.com",
+        "phone_number": "+1 800 555 0199"
+    },
+    "product": {
+        "k": 20.0,
+        "n": 20.0,
+        "p": 20.0,
+        "id": "256b8516-1563-4466-8294-7b654109a770",
+        "npk": "20-20-20",
+        "name": "SuperGrow 20-20-20",
+        "volume": {
+            "unit": "L",
+            "value": "20.8"
+        },
+        "weight": {
```

Comment:
What do you think about formatting weight like:
```json
{
    "volume": {
        "unit": "L",
        "value": "20.8"
    },
    "weight": [
        {
            "unit": "kg",
            "value": 15.0
        },
        {
            "unit": "lb",
            "value": 33.07
        }
    ],
    "density": {
        "unit": "g/cm³",
        "value": "1.2"
    }
}
```

Reply:
`JSON
"product": {
        ...,
        "metrics" : {
            "volume": {
            "unit": "L",
            "value": "20.8"
            },
            "weight": {
                "kg": null,
                "lb": null
            },
            "density": {
                "unit": "g/cm\u00b3",
                "value": "1.2"
            }
        },...
`